apps/roam/src/utils/getDiscourseNodes.ts

```diff
@@ -95,6 +97,11 @@ const getDiscourseNodes = (relations = getDiscourseRelations()) => {
         ),
         graphOverview:
           children.filter((c) => c.text === "Graph Overview").length > 0,
+        githubSync: children.filter((c) => c.text === "GitHub Sync").length > 0,
+        githubCommentsQueryUid: getSubTree({
```

Comment:
Let's nest the comments uid under `githubSync`, just like we do for `canvasSettings`

Reply:
I meant the return should be nested like `canvasSettings`

eg:
```
  canvasSettings: {
    [k: string]: string;
  };
```

So

```
githubSyncConfig = {
  enabled: boolean;
  commentsQueryUid: string;
};
```

Reply:
ah i see. that makes sense.